src/compiler/transformers/module/module.ts

```diff
@@ -1310,7 +1310,7 @@ namespace ts {
 
                     case SyntaxKind.NamedImports:
                         for (const importBinding of namedBindings.elements) {
-                            statements = appendExportsOfDeclaration(statements, importBinding);
+                            statements = appendExportsOfDeclaration(statements, importBinding, /* liveBinding */ true);

```

Comment:
As above, an unconditional live binding will result in code that is invalid in an ES3 target.